sycl/doc/extensions/experimental/sycl_ext_oneapi_bindless_images.asciidoc

```diff
@@ -1712,10 +1721,18 @@ descriptor struct.
 ```cpp
 namespace sycl::ext::oneapi::experimental {
 
+// Types of external semaphore handles
+enum class external_semaphore_handle_type {
+  opaque_fd = 0,
```

Comment:
not: You could maybe write that opaque_fd isn't supported on windows.

Reply:
I've added some minor notes to clarify that `opaque_fd` is only supported on linux based systems. I think going more into depth about which handle types are supported by which operating systems is beyond the scope of this document, and advanced users who want to use the interop APIs will have to know this already when exporting handles from third-party APIs before they even reach SYCL.